SU-SWS/decanter-web
# Exclude local path to scss files in kss info json files

# READY FOR REVIEW

# Summary
- Exclude the local path to the source scss file in kss sections data json files.

# Needed By (Date)
- Whenever

# Urgency
- Not urgent

# Steps to Test

1. Pull this branch and `npm run build`
2. Check that .json files in content/_kss/info do not have user's local path info.

# Affected Projects or Products
- Decanter
